YukkiMusic/plugins/tools/reload.py

```diff
@@ -21,78 +21,61 @@
                                          language)
 from YukkiMusic.utils.formatters import alpha_to_int
 
-### Multi-Lang Commands
+# Multi-Lang Commands
 RELOAD_COMMAND = get_command("RELOAD_COMMAND")
 RESTART_COMMAND = get_command("RESTART_COMMAND")
 
 
-@app.on_message(
-    filters.command(RELOAD_COMMAND)
-    & filters.group
-    & ~filters.edited
-    & ~BANNED_USERS
-)
+@app.on_message((((filters.command(RELOAD_COMMAND) & filters.group)) & ~BANNED_USERS))
 @language
 async def reload_admin_cache(client, message: Message, _):
     try:
         chat_id = message.chat.id
-        admins = await app.get_chat_members(
-            chat_id, filter="administrators"
-        )
+        admins = await app.get_chat_members(chat_id, filter="administrators")
         authusers = await get_authuser_names(chat_id)
         adminlist[chat_id] = []
         for user in admins:
-            if user.can_manage_voice_chats:
+            if user.can_manage_video_chats:
                 adminlist[chat_id].append(user.user.id)
         for user in authusers:
             user_id = await alpha_to_int(user)
             adminlist[chat_id].append(user_id)
         await message.reply_text(_["admin_20"])
-    except:
-        await message.reply_text(
-            "Failed to reload admincache. Make sure Bot is admin in your chat."
-        )
+    except Exception:
```

Comment:
Is there any need to add `Exception` here? you should consider printing the error with `print()`